### DISCUSSION TITLE
Clear existing message before executing emote

### DISCUSSION THREAD

**Discussion by dvicci** [Kiroho/EmoteTome#6] [Suggestions]
I've noticed that if there is any text in the chat window not yet sent, it'll be sent prior to "/emote $emote".  Would it be possible to clear the chat prior to sending "/emote $emote"?

Unsure how this would be accomplished, but perhaps some form of CTRL-A + DEL?

**Comment by Kiroho** [comment]
It should already delete any text from the chat before sending the emote.

Does it always happen, when you have text in your chat or was it a one time thing? 
If the latter, maybe it was just a "hicc up".

**Reply by dvicci** [reply]
Seems to happen all the time.  While I haven't done exhaustive testing with other channels, when in the /say channel, it happens regardless of what text pre-exists, or what emote I attempt to evoke.

Here's an example:

![image](https://github.com/Kiroho/EmoteTome/assets/49494328/f17c0c70-948d-4b70-b1ae-ed972faab6e6)

**Reply by Kiroho** [reply]
That's interesting. 
I tested around for a while, but I couldn't replicate the issue. 

Do you have any other module or software running that interact with your keyboard/input or chat by any chance?
Or a high ping, low fps, an older version of the module or something like this?
